pythonfmu/pythonfmu-export/src/pythonfmu/PySlaveInstance.cpp

```diff
@@ -107,41 +169,39 @@ void PySlaveInstance::clearLogBuffer() const
     PyObject* categoryField = Py_BuildValue("s", "category");
     PyObject* statusField = Py_BuildValue("s", "status");
 
-    if ( pMessages_ != NULL && PyList_Check(pMessages_) ) {
-        auto size = PyList_Size(pMessages_);
-        for (auto i = 0; i < size; i++) {
-            PyObject* msg = PyList_GetItem(pMessages_, i);
+    auto size = PyList_Size(pMessages_);
```

Comment:
This is also a change unrelated to the feature proposed.

Reply:
Removed from PR